### ISSUE TITLE
Issue in training mode : Qt platform plugin xcb

### ISSUE THREAD

**Issue by jcgrenier** [AI-sandbox/gnomix#24]
Hello dear support,

I tried to run my own training set but it's crashing at the re-training step. 

Here's the command line I ran on a cluster machine :
```
python3 ~/bin/gnomix/gnomix.py hap_chr22.vcf chr22.output 22 False genetic_map_chr22_combined_b37.forGnomix.txt ~/References/1000G/ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.noCNV.snv.vcf.gz ~/References/1000G/samples.merged.superpop.map
```
Here are the logs :

std err :
```qt.qpa.xcb: could not connect to display localhost:24.0
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl, xcb.```

stdout :
```Launching in training mode...
Reading vcf file...
Getting genetic map info...
Getting sample map info...
Building founders...
Splitting sample map...
Running Simulation...
Training...
Reading data...
Building model...
Training base models...
Training smoother...
[14:50:58] WARNING: xgboost/src/learner.cc:480:
Parameters: { use_label_encoder } might not be used.

  This may not be accurate due to some parameters are only used in language bindings but
  passed down to XGBoost core.  Or some parameters are not used but slip through this
  verification. Please open an issue if you find above cases.


Evaluating model...
Re-training base models...```

I also have some warnings due to xgboost, but it seems to run alright even with theses:

```/lustre06/project/6005588/shared/virtualenv_python_3.9_gnomix/lib/python3.9/site-packages/xgboost/compat.py:93: FutureWarning: pandas.Int64Index is deprecated and will be removed from pandas in a future version. Use pandas.Index with the appropriate dtype instead.
  from pandas import MultiIndex, Int64Index```
  
Thanks for your help!

Jean-Christophe
